src/App.tsx

```diff
@@ -1,18 +1,130 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useEffect, useState,
+} from 'react';
+import {
+  getTodos,
+  addTodo,
+  CustomTodo,
+  deleteTodo,
+} from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Footer } from './components/Footer/Footer';
+import Form from './components/Form/Form';
+import TodoList from './components/TodoList/TodoList';
+import Warning from './components/Warning/warning';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todosFromTheServer, setTodosFromTheServer] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [todosToShow, setTodosToShow] = useState([...todosFromTheServer]);
```

Comment:
Looks better for me, but it's up to you
```suggestion
  const [visibleTodos, setVisibleTodos] = useState([...todosFromTheServer]);
```